neutron/tests/unit/agent/common/test_ovs_lib.py

```diff
@@ -195,6 +201,11 @@ def test_add_flow(self):
                                  % (lsw_id, vid, ofport))),
             self._ofctl_mock("add-flows", self.BR_NAME, '-',
                              process_input=OFCTLParamListMatcher(
+                                 "hard_timeout=0,idle_timeout=0,cookie=1334,"
+                                 "priority=4,in_port=%s,actions=mod_nw_tos=104"
+                                 % ofport)),
+            self._ofctl_mock("add-flows", self.BR_NAME, '-',
+                             process_input=OFCTLParamListMatcher(
                                  "hard_timeout=0,idle_timeout=0,cookie=1256,"
                                  "priority=4,nw_src=%s,arp,actions=drop"
                                  % cidr)),
```

Comment:
I'm sure this is the wrong place to leave this comment, but here goes:

I think the following tests, or modifications to existing tests, are needed:
  TestOVSAgentQosExtension.test_port_creation_with_dscp
  TestOVSAgentQosExtension.test_port_creation_with_different_dscp_mark(ing?)s

I think the following tests should be modified to include dscp functionality:
  OVSAgentQoSExtensionTestFramework._create_port_with_qos
  TestOVSAgentQosExtension.test_simple_port_policy_update
  TestOVSAgentQosExtension.test_port_qos_disassociation
  TestOVSAgentQosExtension.ttest_policy_rule_delete